scripts/check-equivalent-anchors.py

```
@@ -0,0 +1,72 @@
+#!/usr/bin/env python3
+"""等价白名单锚点漂移检测（六仓统一模式，wop-python-sdk 实例）。
+
+白名单（scripts/mutation_test.py EQUIVALENT_MUTANTS，5 条已论证）仅在变异
```

Ruff (0.16.2)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)